packages/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div
+    v-if="isVisible"
+    :aria-label="title"
+    class="k-modal-fullscreen isOpen"
+    role="dialog"
+    aria-modal="true">
+    <div
+      class="k-modal-dialog"
```

Comment:
I thought we were going to replace all classes of `k-modal-*` to `k-modal-fullscreen-*`.